File: examples/ios/PatAnnExampleObjC/PatAnnExampleObjC/ViewController.m
//
//  ViewController.m
//  PatAnnExampleObjC


#import "ViewController.h"
#import "patann/patann.h"
#import "PatANNExample.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    
    BOOL runAsync = YES;
    
    PatANNExample *example = [PatANNExample new];
    
    if(runAsync) {
        [example runTestAsync];
    } else {
        [example runTestSync];
    }
}


@end

File: examples/ios/PatAnnExampleSwift/PatAnnExampleSwift/ViewController.swift
import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let example = PatANNExample()
            
            // Run synchronous test
        print("Running synchronous test...")
        let syncResult = example.runTestAsync()
        print("Synchronous test \(syncResult ? "passed!" : "failed.")")
    }


}

